src/index.js

```diff
@@ -0,0 +1,181 @@
+class Todo {
+  constructor() {
+    this.tasks = [];
+  }
+
+  save() {
+    localStorage.setItem('localTasks', JSON.stringify(this.tasks));
+  }
+
+  addTask = (desc, completed, index) => {
+    const task = {
+      desc,
+      completed,
+      index,
+    };
+    this.tasks.push(task);
+    this.save();
+    window.location.reload();
+  };
+
+  showBG = () => {
+    const inputForm = document.querySelectorAll('.item');
```

Comment:
- [ ] Hello @joramot, great effort working on the todo list structure, But according to the requirement and the goal of this milestone is that we should create a simple list of todo using `array of an object` and pass in some data that will be rendered dynamically to the DOM.

Example of code snippet 👇 

```JS
const tasks = [
  {
    description: 'Going to the Cinema',
    completed: false,
    index: 1,
  },
  {
    description: 'Coding in the Midnight',
    completed: false,
    index: 2,
  },
  {
    description: 'Up to the task',
    completed: false,
    index: 3,
  },
];
```

then, It has a function that iterates over the tasks array and populates the HTML list item element for each task.

Example of a code snippet 👇 

```JS
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};

```

this is the project requirement 👇 

![func](https://user-images.githubusercontent.com/78634841/236050750-8c9a517e-7479-427c-b704-2e5ed6652110.jpg)

Reply:
Hi, @brainconnect93  yes, what you say is correct, an apology, go ahead with the work a bit, I had not seen that in the next activity, me has to implement the functionality, an apologize, I will make the corrections.